samples/06.i.direct-line-speech/README.md

```diff
@@ -0,0 +1,197 @@
+
+After the Azure resource is created, configure your bot with a new Direct Line Speech channel and associate the channel with your newly created Azure resource. You can follow [this article for connecting your bot to Direct Line Speech channel](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-channel-connect-directlinespeech?view=azure-bot-service-4.0).
+
+You will also need to update your bot to use 4.6 SDK. You can follow [this article for upgrading your bot to use Direct Line Speech](https://docs.microsoft.com/en-us/azure/bot-service/directline-speech-bot?view=azure-bot-service-4.0).
+
+### Using subscription key
+
+We will start by using subscription key to connect to Direct Line Speech channel. Subscription key is not recommended to use in production environment. In this sample, we will use it to verify if our configuration is correct. In our latter section, we will upgrade the sample to use authorization token instead.
+
+We are replacing the existing code by creating an adapter set using the new `createDirectLineSpeechAdapters` function. We will pass the adapter set to Web Chat using [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax).
+
+```diff
+- const res = await fetch('https://webchat-mockbot.azurewebsites.net/directline/token', { method: 'POST' });
+- const { token } = await res.json();
++ const adapters = await window.WebChat.createDirectLineSpeechAdapters({
++   fetchCredentials: {
++     region: 'westus2',
++     subscriptionKey: 'YOUR_COGNITIVE_SERVICE_KEY'
++   }
++ });
+
+  window.WebChat.renderWebChat(
+    {
+-     directLine: window.WebChat.createDirectLine({ token })
++     ...adapters
+    },
+    document.getElementById('webchat')
+  );
+```
+
+> For list of supported options, you can refer to our [README on Direct Line Speech SDK](https://github.com/microsoft/BotFramework-WebChat/tree/master/packages/directlinespeech/).
+
+Now, you should be able to communicate with your bot using the Direct Line Speech channel. If you encounter any problems while connecting to the service, you should use the in-browser developer tool and investigate.
+
+### Using authorization token
+
+After you verified Web Chat is communicating with your bot using Direct Line Speech channel, we will upgrade the code to use authorization token instead.
+
+The authorization token is generated using subscription key. To secure the token, the exchange must be done on server-side and the subscription key should never be passed to the browser.
+
+First, create an async function `fetchCredentials`. When called, it will fetch the authorization token from the server. In this sample, we will fetch the token from our demo bot. The function will return `authorizationToken` and `region` in a JavaScript object.
```

Comment:
```suggestion
First, create an async function `fetchCredentials`. When called, it will fetch the authorization token from the server. In this sample, the token is fetched from the demo bot, Mock Bot. The function will return `authorizationToken` and `region` in a JavaScript object.
```

Reply:
I know, we should use less "we will" in education part of sample.